.travis.yml

```diff
@@ -1,7 +1,17 @@
 language: node_js
-install:
-  - yarn
+node_js:
+  - 'lts/*'
```

Comment:
Since EDS runs on node v10.16.3, we probably shouldn't be using cutting edge `lts/*`
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc
If these are deleted, Travis would read `.nvmrc` file and know to use a node version v10.16.3

Reply:
fixed